robowflex_library/src/geometry.cpp

```diff
@@ -172,3 +195,28 @@ const shape_msgs::Mesh Geometry::getMeshMsg() const
 
     return boost::get<shape_msgs::Mesh>(msg);
 }
+
```

Comment:
+1, was just comparing `robowflex::Geometry` to `vicon_object_server::Geometry`, and the latter has a lot of helper functions like this that we need.